install.md

```diff
@@ -37,13 +37,28 @@ cd conmon
 make
 ```
 
+## Install with Ansible
+
+An [Ansible Role](https://github.com/alvistack/ansible-role-conmon) is also available for automate above steps:
```

Comment:
nit:  "for automate" -> "to automate the"

Reply:
fixed ;-)